docs/pages/introduction/why-not-expo.mdx

```diff
@@ -3,108 +3,110 @@ title: Limitations
 sidebar_title: Limitations
 ---
 
-import { InlineCode } from '~/components/base/code';
 import { Collapsible } from '~/ui/components/Collapsible';
 
-Your success will be limited if you don't know the limitations of your tools. A good software engineer strives to understand the tradeoffs in the decisions they make.
+Your success will be limited if you don't know the limitations of your tools. A good software engineer strives to understand the tradeoffs in their decisions.
```

Comment:
```suggestion
We believe Expo tools are a great option for just about anyone looking to create amazing cross-platform apps quickly. Nonetheless, everyone's project is unique, and there may be reasons Expo is not the right choice for what you're building. Read on for some reasons why you may _not_ want to use Expo tools.
```
The original commentary about what is or isn't a "good developer" feels out-of-character with the voice we've been striving for with docs more recently. Just a suggestion above of a possible voicing that's a little more positive.

Reply:
Thanks for this update! Definitely sounds more positive! 🙏 

I'll just make a small to remove the word "just about" which I'll update on my own.